.ci/azure/pipelines/release.yml

```diff
@@ -20,8 +20,8 @@ jobs:
     displayName: 'Install Docker'
 
   - bash: |
-      sudo apt-get update -y -qq
-      sudo apt-get install -y git
-      git config --global user.email "azure-pipeline@kata.io"
-      git config --global user.name "azure-pipeline"
-    displayName: 'Setup'
+      git clone https://github.com/egernst/kata-deploy.git
```

Comment:
I thought we have a copy of this in packaging and we were building from that? Not true?

Reply:
Yeah, let me close this and rework the changes locally.